plugins/source/postgresql/client/cdc.go

```diff
@@ -226,31 +226,6 @@ func (c *Client) listenCDC(ctx context.Context, res chan<- message.SyncMessage)
 				}
 				res <- resource
 			case *pglogrepl.DeleteMessage:
-				rel, ok := relations[logicalMsg.RelationID]
```

Comment:
Is there an issue why we're removing this?

Reply:
Because it doesn't do anything